lib/Runtime/Base/ThreadContext.cpp

```diff
@@ -3230,6 +3230,53 @@ ThreadContext::InvalidateProtoInlineCaches(Js::PropertyId propertyId)
     }
 }
 
+void
+ThreadContext::InvalidateMissingPropertyInlineCaches(const Js::Type *type, Js::PropertyId propertyId) {
+    InlineCacheList *inlineCacheList;
+    if (protoInlineCacheByPropId.TryGetValue(propertyId, &inlineCacheList))
+    {
+        if (PHASE_TRACE1(Js::TraceInlineCacheInvalidationPhase))
+        {
+            Output::Print(_u("InlineCacheInvalidation: invalidating missing-property proto caches for property %s(%u), type %d\n"),
```

Comment:
Question for reviewers: I'd like to include the type name as well as its ID in the tracing message, but I couldn't find an API that provided access to that information.  Does such a thing exist?  (Given type sharing, this may not be feasible.)

Reply:
Do you mean the name of the type handler? There's a virtual DynamicTypeHandler::Dump method that spews out the whole thing, but I don't see a DBG method that gives back just the name. Fine with me if you want to add one.